lib/page/house_wikidata.rb

```diff
@@ -12,21 +13,49 @@ def title
       "EveryPolitician: #{country.name} — #{house.name}"
     end
 
-    def country
-      index.country(country_slug)
+    def house_name
+      house.name
     end
 
-    def house
-      country.legislature(house_slug)
+    def country
+      index.country(country_slug)
     end
 
     # TODO: we shouldn't be passing raw Popolo, only what's needed
     def popolo
       house.popolo
     end
 
+    def people_with_wikidata
```

Comment:
This can all be massively simplified by using `partition`, as the existing template already does. It will divide an array into two separate arrays, based on some condition (e.g. whether the entries have wikidata or not!)